.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,7 @@
 
 #### Summary
 <!--
-A description of what this pull request does.
+A description of what this pull request does, as well as QA test steps (if applicable).
```

Comment:
Should we mention that the JIRA ticket is the primary place to put QA test steps?

Reply:
I added a note. QA doesn't mind having it in the PR description since most of the testing is done on PRs.